build.gradle

```diff
@@ -85,10 +93,10 @@ dependencies {
     testImplementation 'com.squareup.okhttp3:okhttp:4.11.0'
 }
 
-// Creates a version.txt file containing the current version of the SDK.
-// This file is picked up and parsed by our Ship Orb to determine the version.
-task exportVersion()  {
-    doLast {
-        new File(rootDir, "version.txt").text = "$version"
+
+jar {
+    // Will include dependencies to create fat jar
+    from {
+        configurations.runtimeClasspath.collect { it.isDirectory() ? it : zipTree(it) }
     }
```

Comment:
Is this a leftover change from some debugging?

Reply:
Yep, I have removed it now. My bad!